src/main/java/org/jsoup/helper/Validate.java

```diff
@@ -119,4 +121,22 @@ public static void wtf(String msg) {
     public static void fail(String msg) {
         throw new IllegalArgumentException(msg);
     }
+
+    /**
+     * Validate the connection
+     * @param url url needs to be connected
+     * @return if the connection is OK
+     */
+    public static boolean check_connection(String url) {
```

Comment:
Why would this be a new public contract API? If we were to use it, would be better in a test helper class.

And please use normal Java style guide. This is not Python.